release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -88,6 +88,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-multiple-cidr"]
+==== Define multiple CIDR blocks for one network security group (NSG) rule
+
+With this release, IP addresses and ranges are handled more efficiently in NSGs for {product-title} clusters hosted on Azure. As a result, the maximum limit of CIDRs for all Ingress controllers in Azure clusters, using the `allowedSourceRanges` field, increases from approximately 1000 to 4000 CIDRs.
```

Comment:
```suggestion
With this release, IP addresses and ranges are handled more efficiently in NSGs for {product-title} clusters hosted on Azure. As a result, the maximum limit of CIDRs for all Ingress Controllers in Azure clusters, using the `allowedSourceRanges` field, increases from approximately 1000 to 4000 CIDRs.
```

Reply:
Should "Ingress controllers" be "Ingress Controllers"? Per https://redhat-documentation.github.io/supplementary-style-guide/#ingress-controller

Reply:
yes, good catch!